CleverFerret/src/main/java/com/universalmedialibrary/services/metadata/covers/AppleBooksCoverSource.kt

```
@@ -0,0 +1,108 @@
+            
+            val coverResult = CoverResult(
+                url = hdUrl,
+                width = 2000,
+                height = 2000,
+                quality = CoverQuality.ULTRA_HD,
+                source = sourceName,
+                sourceId = book.optString("trackId")
+            )
+            
+            Result.success(coverResult)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)